src/signalrclient/connection_impl.cpp

```diff
@@ -155,8 +155,64 @@ namespace signalr
         std::weak_ptr<connection_impl> weak_connection = shared_from_this();
         const auto& token = m_disconnect_cts;
 
+		const auto started = [weak_connection, callback, token](std::shared_ptr<transport> transport, std::exception_ptr exception)
```

Comment:
I don't see why this was moved, could we revert it if it's unrelated to the change.

Reply:
To be captured in the lambda.

Reply:
Oh I see, I missed that it was being used in two places.
```suggestion
        const auto transport_started = [weak_connection, callback, token](std::shared_ptr<transport> transport, std::exception_ptr exception)
```